client/src/components/Register/Register.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Button, Box, TextField, Typography } from '@mui/material'
+import { useAuth } from '../Auth/AuthProvider'
+import { Navigate, useNavigate, useLocation } from 'react-router-dom'
```

Comment:
Why import useLocation if you're not going to use it?

Reply:
useLocation removed